kvm/src/lib.rs

```diff
@@ -1474,7 +1518,7 @@ mod tests {
         assert_eq!(get_raw_errno(faulty_vm_fd.create_vcpu(0)), badf_errno);
         let faulty_vcpu_fd = VcpuFd {
             vcpu: unsafe { File::from_raw_fd(-1) },
-            run_mmap: MemoryMapping::new(10).unwrap(),
+            kvm_run_ptr: KvmRunWrapper::new(10),
```

Comment:
```suggestion
            kvm_run_ptr: KvmRunWrapper {
                kvm_run_ptr: 0x0 as *mut u8,
            },
```
There is no need to implement another KvmRunWrapper given that this is concerned only with a faulty vcpu fd.